src/scss/elements/_product-card.scss

```diff
@@ -0,0 +1,104 @@
+.product-card {
+  width: 198px;
+  background-color: #fff;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+
+  &__container {
+    width: 166px;
+    padding-top: 32px;
+    padding-inline: 16px;
+    padding-bottom: 16px;
+  }
+
+  &__image {
+    position: relative;
```

Comment:
```suggestion

```

Reply:
Sorry, I didn't quite understand, what I need to fix in 12-15 lines and 21-23, 25 lines